nasa-gibs/worldview
# WV-2374: Creating React Components for Map UI

## Description

This update replaces the files previously called `ui.js` and `combine-ui.js` with React components in the new MapUI folder. The behavior of WV should be entirely the same. Changing these into React components will make the logic in these files much easier to maintain and read when it requires changes or updates.

## How To Test

- Checkout to branch `wv-2445`
- `npm run watch`
- Test as much functionality as you can to make sure that WV still operates entirely the same. The scope of these files is very wide ranging and includes many aspects of WV such as updating projections, dates, markers, layers, opacity etc...
